nle/env/base.py

```diff
@@ -313,21 +117,19 @@ def __init__(
             "chars",
             "colors",
             "specials",
-            "status",
+            "blstats",
             "message",
-            "inventory",
```

Comment:
bye bye inventory

Reply:
We can re-add the inventory observation once we know what to do with it.

Sadly, this isn't the only "menu" we'll have to deal with: I noticed NetHack will automatically pop up an ingame-window for tiles with lots of objects on them, and there's no way to make it list that with topline messages (although there is https://nethackwiki.com/wiki/Options#pile_limit). So to deal with objects, we will have to think about what to do there.